web_timeline/README.rst

```diff
@@ -0,0 +1,140 @@
+  contains the end of the event.
+* default_group_by (required): it defines the name of the field that will be
+  taken as default group by when accessing the view or when no other group by
+  is selected.
+* event_open_popup (optional): when set to true, it allows to edit the events
+  in a popup. If not (default value), the record is edited changing to form
+  view.
+* colors (optional): it allows to set certain specific colors if the expressed
+  condition (JS syntax) is met.
+
+You also need to declare the view in an action window of the involved model.
+
+Example:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <record id="view_task_timeline" model="ir.ui.view">
+            <field name="model">project.task</field>
+            <field name="type">timeline</field>
+            <field name="arch" type="xml">
+                <timeline date_start="date_start"
+                          date_stop="date_end"
+                          string="Tasks"
+                          default_group_by="user_id"
+                          event_open_popup="true"
+                          colors="#ec7063:user_id == false;#2ecb71:kanban_state=='done';">
+                </timeline>
+            </field>
+        </record>
+
+        <record id="project.action_view_task" model="ir.actions.act_window">
+            <field name="view_mode">kanban,tree,form,calendar,gantt,timeline,graph</field>
+        </record>
+    </odoo>
+
+Usage
+=====
+
+For accessing the timeline view, you have to click on the button with the clock
+icon in the view switcher. The first time you access to it, the timeline window
+is zoomed to fit all the current elements, the same as when you perform a
+search, filter or group by operation.
+
+You can use the mouse scroll to zoom it or out in the timeline, and make click
+on any free area and drag for panning the view in that direction.
+
+The records of your model will be shown as rectangles whose widths are the
+duration of the event according our definition. You can select them clicking
+on this rectangle. You can also use Ctrl or Shift keys for adding discrete
+or range selections. Selected records are hightlighted with a different color
+(but the difference will be more noticeable depending on the background color.
+Once selected, you can drag and move the selected records across the timeline.
+
+When a record is selected, a red cross button appears on the upper left corner
+that allows to remove that record. This doesn't work for multiple records
+although they were selected.
+
+Records are grouped in different blocks depending on the group by criteria
+selected (if none is specified, then the default group by is applied).
+Dragging a record from one block to another change the corresponding field to
+the value that represents the block. You can also click on the group name to
+edit the involved record directly.
+
+Making double click on the selected record, you would be able to edit it.
```

Comment:
Double-click on the record to edit it.
Double-click in open area to create a new record with the group and start date linked to the area you clicked in.